### DISCUSSION TITLE
Integrazione Alexa su home assistant

### DISCUSSION THREAD

**Discussion by blackeaglece** [indomus/forum#1346] [HUB: Home Assistant]
Ciao ho provato più volte a seguire la guida dell'integrazione di alexa su HA, mi va tutto bene, il test mi da ok. Quando vado nell'applicazione, visualizzo la skill haaska e mi dice che devo collegarla all'account per abilitarla all'uso... è qui ho il problema. Mi risponde sempre : `"ci dispiace, si è verificato un errore etc etc"` quando clicco su visualizza i dettagli mi esce quest'errore:
`400 bad request
The redirect URI you provided has not been whitelisted for your application. Please add your redirect URI in the allowed Return etc etc`
Ho cancellato e rifatto procedura 4-5 volte, eseguito tutto riga per riga ma ricevo sempre lo stesso errore. Come posso fare?
Grazie a tutti

**Comment by snakuzzo** [comment]
Come hai configurato HA ?
hai un reverse proxy a monte ?

**Reply by blackeaglece** [reply]
> Come hai configurato HA ? hai un reverse proxy a monte ?

Ciao, ho HA in docker su raspi 4, collegato in wan tramite rj45 con ip fisso. Dall'esterno (https://miacasa.duckdns.org:8123) lo raggiungo senza problemi
ps: non ho un reverse proxy

**Reply by snakuzzo** [reply]
Più di dirti di riverificare cosa hai messo nel `config.json` sulla lambda function su AWS...non saprei
